wrapper/CSharp/wolfTPM-tests.cs

```diff
@@ -540,6 +540,8 @@ public void TryGenerateCSRCustomOID()
             Assert.That(rc, Is.GreaterThan(0));
 
             Console.WriteLine("CSR PEM {0} bytes", rc.ToString());
+            var writer = new BinaryWriter(File.OpenWrite("csr.pem"));
```

Comment:
This could be handy. Any issue running it over and over?

Reply:
Glad you like it.!

If you used over and over then it would probably write all the things to the same file?  I would think the writer needs to fall out of scope and then the garbage collector would then close the file?

I can break this out to its own function.  Then it could fall out of scope and it would make things clearer.